keyboards/jian/README.md

```diff
@@ -0,0 +1,16 @@
+# Jian
```

Comment:
Could you rename this file so it is lower case? 

We have a mix of operating systems used, with various levels of case sensitivity.  Keeping everything lowercase makes things simpler.